### DISCUSSION TITLE
Delete relationship

### DISCUSSION THREAD

**Discussion by neopaf** [structurizr/java#292] [Q&A]
### Question

I wanted to "polish" workspace from workspace-level script.
Activated it at the end of `workspace.dsl`.
Located relevant relationship.
Not sure how to properly remove it.

Please give some pointer as to how having a `relationship` object instance to properly remove it?

**Comment by neopaf** [comment]
?

**Comment by neopaf** [comment]
?

**Comment by simonbrowndotje** [comment]
There's currently no way to remove an individual relationship, but perhaps [workspace.trim()](https://github.com/structurizr/java/blob/master/structurizr-core/src/main/java/com/structurizr/Workspace.java#L169) would help?

**Comment by neopaf** [comment]
Looks good. Thanks!
